app/controllers/proposal_controller.rb

```diff
@@ -16,6 +16,8 @@ def show
   def new
     @user = User.new
     @url = conference_program_proposal_index_path(@conference.short_title)
+    @has_language = @program.languages.present?
```

Comment:
I defined @has_language because I also need to know if languages is present in the view, but I can change it.
I forgot the :edit action :frowning: